src/main/java/com/microsoft/azure/msalwebsample/BasicConfiguration.java

```diff
@@ -19,6 +19,7 @@
     private String redirectUriSignin;
     private String redirectUriGraph;
     private String secretKey;
+    private String msGraphEndpointHost;
```

Comment:
Shouldn't this key be "graphEndpointHost" to match the key in the application.properties below "aad.graphEndpointHost".

Reply:
Yes you're right. Updated to aad.msGraphEndpointHost;